config/config.go

```diff
@@ -56,5 +68,71 @@ func (c *Config) initConfigFromFile(path string) error {
 	if c.HaProxyPidFile == "" {
 		return errors.New("haproxy_pid_file is required")
 	}
+
+	if c.BackendTLS.CACertificatePath != "" {
+		pemData, err := os.ReadFile(c.BackendTLS.CACertificatePath)
```

Comment:
❓ Is there a routing-release PR to go with this PR? 

I am curious to see if CACertificatePath would ever be empty? I assume the path would always be filled, but the file might be empty? But it depends on routing-release config implementation.

Reply:
ah, maybe it doesn't matter, I see below that the errors are only thrown if there is actual data in the file. I see. nvm.

Reply:
op I found it https://github.com/cloudfoundry/routing-release/pull/422/files

Reply:
Note: In the PR the path _is_ set conditionally if a cert is provided